packages/@sanity/default-layout/src/components/RenderTool.js

```diff
@@ -1,28 +1,101 @@
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import React from 'react'
 import tools from 'all:part:@sanity/base/tool'
+import Button from 'part:@sanity/components/buttons/default'
+import ErrorBorkImage from './ErrorBorkImage'
+import styles from './RenderTool.css'
+
+class RenderTool extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {error: null, showErrorDetails: __DEV__}
```

Comment:
Unless we don't need anything from props to set initial state, I think it's better to set initial state using class field declarations, e.g. like we do here: https://github.com/sanity-io/sanity/blob/9fdd9ed6c2c318bbae8bbbf097e3530afebc2478/packages/%40sanity/default-layout/src/components/DefaultLayout.js#L40-L45